crypto/bip39.c

```diff
@@ -154,13 +154,32 @@ int mnemonic_to_entropy(const char *mnemonic, uint8_t *entropy) {
   if (bi != n * 11) {
     return 0;
   }
-  memcpy(entropy, bits, sizeof(bits));
+  memcpy(bits, result, sizeof(result));
+
+  // returns amount of entropy + checksum BITS
   return n * 11;
 }
 
+int mnemonic_to_entropy(const char *mnemonic, uint8_t *entropy) {
```

Comment:
I realized tests to cover this refactored function are currently missing, and if we decide to keep it, I will add them. It's worth noting though that this function is currently not actually used anywhere, it's there rather for the sake of completeness, so we may as well just drop it completely